DMCompiler/Compiler/DM/DMLexer.cs

```diff
@@ -7,30 +7,25 @@
 namespace DMCompiler.Compiler.DM {
     public partial class DMLexer : TokenLexer {
         public static List<string> ValidEscapeSequences = new() {
-            "t", "n",
-            "[", "]",
-            "\\", " ", "\"", "'",
-            "<", ">",
-
             "icon",
             "Roman", "roman",
             "The", "the",
-            "A", "a", "An", "an",
+            "A", "a",
+            "An", "an",
```

Comment:
shouldn't these be reversed?

Reply:
Doesn't matter, the main change is how the escapes are read in to match to the list, and that part only uses Contains